internal/spicedbx/schema.go

```diff
@@ -108,24 +54,205 @@ func GenerateSchema(namespace string, resourceTypes []types.ResourceType) (strin
 	return out.String(), nil
 }
 
-// GeneratedSchema generated the schema for a namespace
+// GeneratedSchema generates the schema for a namespace
 func GeneratedSchema(namespace string) string {
-	resourceTypes := []types.ResourceType{
-		{
-			Name: "loadbalancer",
-			Actions: []string{
-				"get",
-				"update",
-				"delete",
-			},
-			TenantActions: []string{
-				"create",
-				"list",
+	policyDocument := iapl.PolicyDocument{
+		ResourceTypes: []iapl.ResourceType{
+			{
+				Name:     "role",
+				IDPrefix: "idenrol",
+				Relationships: []iapl.Relationship{
+					{
+						Relation:       "subject",
+						TargetTypeName: "subject",
+					},
+				},
+			},
+			{
+				Name:     "user",
+				IDPrefix: "idenusr",
+			},
+			{
+				Name:     "client",
```

Comment:
whats a client?

Reply:
Client here refers to an OAuth 2.0 client. Once this is more mature we'll have a YAML file that defines the policy instead of hardcoding things here, with the exception of roles.